core/src/test/java/org/apache/calcite/test/fuzzer/RexProgramFuzzyTest.java

```diff
@@ -65,7 +65,7 @@
 
   private static final Duration TEST_DURATION =
       Duration.of(Integer.getInteger("rex.fuzzing.duration", 5), ChronoUnit.SECONDS);
-  private static final long TEST_ITERATIONS = Long.getLong("rex.fuzzing.iterations", 18);
+  private static final long TEST_ITERATIONS = Long.getLong("rex.fuzzing.iterations", 2000);
   // Stop fuzzing after detecting MAX_FAILURES errors
```

Comment:
Could you explain the reason behind this change?

Reply:
yes; the fuzzytest would have been able to catch this issue earlier - but it was only doing 18 tests - and that was not enough...I think it makes sense to raise the bar...so it will have more opportunities to keep us safe from issues like this.

running 2000 takes <2 sec on my machine ; and so it could still fit into the 5sec timewindow it has